jellyfin-api/src/main/kotlin/org/jellyfin/sdk/api/client/KtorClient.kt

```diff
@@ -104,12 +104,14 @@ public open class KtorClient(
 		queryParameters: Map<String, Any?> = emptyMap(),
 		requestBody: Any? = null,
 	): Response<T> {
-		val logger = LoggerFactory.getLogger(this::class.java)
 		val url = createUrl(pathTemplate, pathParameters, queryParameters)
 
 		// Log HTTP call with access token removed
-		val safeUrl = accessToken?.let { url.replace(it, "******") } ?: url
-		logger.info("$method $safeUrl")
+		val logger = KotlinLogging.logger {}
```

Comment:
What's the reason for having the logger within the method here?

Reply:
To not expose the logger outside of the inline function